src/main/java/seedu/address/model/tag/Attendance.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model.tag;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import seedu.address.model.person.AttendanceStatus;
+
+/**
+ * Represents a Tag in the address book.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidDate(String)} (String)}
+ */
+public class Attendance {
+
+    public static final String MESSAGE_CONSTRAINTS = "Attendance date should be in dd-MM-yyyy";
```

Comment:
should ``MM``` follow small caps since ```dd``` and ```yyyy``` are also small caps

Reply:
i think it is because lowercase mm stands for minutes, and uppercase MM stands for months